app/(main)/layout.tsx

```diff
@@ -51,7 +51,7 @@ const RootLayout = ({
                 </a>
               </div>
             </main>
-            <Toaster />
+            <Toaster position="top-right" containerStyle={{ top: '4rem' }} />
```

Comment:
Does `top: '4rem'` ensure it clears the navbar?

Reply:
Yes, the navbar has a height of `4 rem`. I could increase the offset to give it some padding between the navbar and the toast though.

<img width="1457" alt="image" src="https://github.com/user-attachments/assets/6afb78f3-912e-468f-95ec-59bd5a7c16dc">

Reply:
@kepsteen I like that idea, just so it's not butting up against the bottom of the navbar. It looks fine currently because of the gradient on the navbar, but if we change that we'd need to add that padding.

Reply:
I see what you mean. I'll add more padding